modules/nf-core/ensemblvep/vep/main.nf

```diff
@@ -31,6 +31,7 @@ process ENSEMBLVEP_VEP {
     def file_extension = args.contains("--vcf") ? 'vcf' : args.contains("--json")? 'json' : args.contains("--tab")? 'tab' : 'vcf'
     def compress_cmd = args.contains("--compress_output") ? '' : '--compress_output bgzip'
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def stats_cmd = args.contains("--no_stats") ? '' : "--stats_file ${prefix}.summary.html"
```

Comment:
Is this an officially supported parameter?

Reply:
I'd rather add the `"--stats_file ${prefix}.summary.html` to the ext.args

Reply:
Sounds good! I will remove it 👍🏻